src/lib/ui/thermostat/AccordionRow.tsx

```diff
@@ -0,0 +1,44 @@
+import { type PropsWithChildren, useState } from 'react'
+
+import { ChevronWideIcon } from 'lib/icons/ChevronWide.js'
+
+interface AccordionRowProps {
+  title: string
+  triggerRightContent?: JSX.Element
+}
+
+export function AccordionRow({
+  title,
```

Comment:
Should we avoid using `title` as a prop name since it conflict with the `title` attribute (unless it is actually passed through to some title attribute)?

We are using `heading` elsewhere to avoid this.

Reply:
Ahh yes, thanks for pointing this out!

Reply:
Do you think `heading` is a bit suggestive here? Would `label` work or do you just prefer `heading`?

Reply:
Label works too. All the options have some corresponding  html thing, `title` has an attribute / tag, `heading` = h1,h2,..., `label` has `<label>`. The main reason we avoided `title` is actually because we need our props to never conflict with attributes since they are make into custom elements. That rule doesn't really apply to internal components. So keeping title is would be ok. We just have to pick something that maps closely to the intent and not introduce too many different names in the code for the "same thing" across our internal ui lib.

Reply:
Yeah totally—there's so much overlap! I think `label` actually makes the most sense in this case. I'll do that.